rcgen/tests/botan.rs

```diff
@@ -17,12 +18,12 @@ fn default_params() -> (CertificateParams, KeyPair) {
 	(params, key_pair)
 }
 
-fn check_cert(cert_der: &[u8], cert: &Certificate) {
+fn check_cert(cert_der: &CertificateDer, cert: &Certificate) {
```

Comment:
If you decide it's worth having the explicit lifetimes here and L26 could use the same treatment.